src/build/analyzer.ts

```diff
@@ -149,46 +159,75 @@ export class StreamAnalyzer extends Transform {
       return {
         depsToFragments,
         fragmentToDeps,
+        fragmentToFullDeps,
       };
     });
   }
-
+  _collectScriptsAndStyles(tree: DocumentDescriptor): DocumentDeps {
```

Comment:
cal you move these to below _getDependencies? I like to order methods roughly by call order if possible